pkg/runtime/wfengine/wfengine.go

```diff
@@ -63,20 +58,23 @@ func IsWorkflowRequest(path string) bool {
 	return backend.IsDurableTaskGrpcRequest(path)
 }
 
-func NewWorkflowEngine(appID string, spec config.WorkflowSpec) *WorkflowEngine {
+func NewWorkflowEngine(appID string, spec config.WorkflowSpec, backendManager processor.WorkflowBackendManager) *WorkflowEngine {
 	engine := &WorkflowEngine{
-		spec:          spec,
-		actorsReadyCh: make(chan struct{}),
+		spec: spec,
 	}
-	be := NewActorBackend(appID)
-	engine.backend = be
 
-	return engine
-}
+	var backendType string
+	backendComponentInfo, ok := backendManager.WorkflowBackendComponentInfo()
+	if ok && backendComponentInfo != nil {
+		backendType = backendComponentInfo.WorkflowBackendType
+	} else {
+		backendType = ActorBackendType
+	}
 
-// GetInternalActorsMap returns a map of internal actors that are used to implement workflows
-func (wfe *WorkflowEngine) GetInternalActorsMap() map[string]actors.InternalActor {
-	return wfe.backend.GetInternalActorsMap()
+	be := InitializeWorkflowBackend(appID, backendType, engine, backendComponentInfo, wfBackendLogger)
```

Comment:
Do we have an info-level log statement anywhere indicating which backend we selected?

Reply:
Added info log.